Gruntfile.js

```diff
@@ -132,6 +132,25 @@ module.exports = function(grunt) {
         }
     };
 
+    let webpackDevConfig = Object.assign({}, webpackConfig, {
+        resolve:
+            alias = {
```

Comment:
Not sure how this worked: the syntax is invalid, I've fixed it in the next commit after merge.

Reply:
Argh, and it still doesn't work, because I missed the modules key inside resolve and Object.assign just overwrites the resolve key. Will send a PR for that.